extract_project_descriptions_and_public_interests.py

```diff
@@ -0,0 +1,75 @@
+""" This file is for getting the project descriptions and 
+public interests from xml and save them to a csv file. """
+import time
+import re
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+def extract_project_description_and_public_interest(url_df):
```

Comment:
I don't know type of url_df: is it a list of urls?
`def extract_project_description_and_public_interest(url_df: list[str]):`